main.go

```diff
@@ -146,6 +146,9 @@ func mainError() error {
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CAFile, "", "Certificate authority file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.CrtFile, "", "Certificate file path to use to authenticate with Kubernetes.")
 	daemonCommand.PersistentFlags().String(f.Service.Kubernetes.TLS.KeyFile, "", "Key file path to use to authenticate with Kubernetes.")
+	daemonCommand.PersistentFlags().Bool(f.Service.Tenant.Ignition.Debug.Enabled, false, "Enable services which help debugging ignition.")
```

Comment:
I had the idea for `ignition-operator` to support fields like this in there and have debug units there. Then we could have e2e tests running with provider operators and ignition operator running. The provider operator wouldn't need modification while ignition operator injects the logging unit.

Not blocking here - just sharing my ideas.

Reply:
I understand this.

Before this, I worked in the direction on support additions in form of separate ignition configs (https://github.com/giantswarm/azure-operator/tree/ignition_additions)
And it works fine, but in case of tests it is hard to pass my ignition config into container for tests. So I left that idea